tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Teams.Tests/ActionTests.cs

```diff
@@ -368,6 +369,31 @@ public async Task Action_SendTaskModuleUrlResponseError()
             await TestUtils.RunTestScript(_resourceExplorerFixture.ResourceExplorer);
         }
 
+        [Fact]
```

Comment:
> [Fact] [](start = 7, length = 7)

Would be better to have the Teams tests in a separate class as the Teams Actions live in a separate project.

Reply:
This is the Microsoft.Bot.Builder.Dialogs.Adaptive.Teams.Tests project